app/Routes.hs

```diff
@@ -26,21 +54,22 @@ strictRoutes aboutContents privacyContents = [
                           findAndSavePrereqsResponse),
     ("image", look "JsonLocalStorageObj" >>= graphImageResponse),
     ("timetable-image", lookText' "session" >>= \session -> look "courses" >>= exportTimetableImageResponse session),
-    ("timetable-pdf", look "courses" >>= \courses -> look "JsonLocalStorageObj" >>= exportTimetablePDFResponse courses),
+    ("timetable-pdf", look "courses" >>= \coursesList -> look "JsonLocalStorageObj" >>= exportTimetablePDFResponse coursesList),
     ("post", retrievePost),
     ("post-progress", postResponse),
     ("draw", drawResponse),
     ("about", aboutResponse aboutContents),
     ("privacy", privacyResponse privacyContents),
-    ("course", lookText' "name" >>= retrieveCourse),
-    ("all-courses", liftIO allCourses),
-    ("graphs", liftIO queryGraphs),
-    ("course-info", lookText' "dept" >>= courseInfo),
-    ("depts", liftIO deptList),
+    ("graphs", liftIO graphs),
     ("timesearch", searchResponse),
     ("generate", generateResponse),
-    ("calendar", look "courses" >>= calendarResponse),
     ("get-json-data", lookText' "graphName" >>= \graphName -> liftIO $ getGraphJSON graphName),
+    
+    ("course", lookText' "name" >>= retrieveCourse),
+    ("courses", liftIO courses),
```

Comment:
The renaming is good, but this change is suspicious because there's no other place where the route is renamed. Either you missed something or this route isn't being used at all (and can be removed).